e2e-tests/development-runtime/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This was necessary to have correct GraphQL errors. With our default ESLint config we catch GraphQL errors (or at least all the ones I tested) and output it as an ESLint error (compile error) and not as GraphQL errors.

Another side-effect of this now is that we test that our default config is merged with any custom one as the tests for the ESLint checks here still pass